common/src/main/java/com/viaversion/viaversion/protocols/protocol1_16to1_15_2/packets/InventoryPackets.java

```diff
@@ -45,6 +47,16 @@ public InventoryPackets(Protocol1_16To1_15_2 protocol) {
 
     @Override
     public void registerPackets() {
+        // clear cursor item to prevent client to try dropping it during navigation between multiple inventories causing arm swing
+        PacketHandler cursorRemapper = wrapper -> {
+            PacketWrapper clearPacket = wrapper.create(ClientboundPackets1_16.SET_SLOT);
+            clearPacket.write(Type.BYTE, (byte)-1);
```

Comment:
the type was changed 1/2 days ago
```suggestion
            clearPacket.write(Type.UNSIGNED_BYTE, (byte)-1);
```